src/components/category-item-card/CategoryItemCard.jsx

```diff
@@ -0,0 +1,29 @@
+import { Card, CardContent, Typography } from '@mui/material'
+import { Link, useNavigate } from 'react-router-dom'
+import { styles } from '~/components/category-item-card/CategoryItemCard.styles'
+
+const CategoryItemCard = ({ icon, title, offerCount, to }) => {
+  const navigate = useNavigate()
+
+  const handleCardClick = () => {
+    navigate(to)
+  }
+
+  return (
+    <Card component={Link} onClick={handleCardClick} sx={styles.card} to={to}>
```

Comment:
Here `to` and `navigate` are used. Can we use only `onClick` without `to` and `Link`?

Reply:
Okay good notice, I did it!